react/features/face-landmarks/actions.ts

```diff
@@ -1,11 +1,11 @@
-// @flow
-import 'image-capture';
```

Comment:
Is the polyfill not needed anymore?

Reply:
Seemed to work without it, but I'll add it back just to be sure